### DISCUSSION TITLE
Conda install fails

### DISCUSSION THREAD

**Discussion by tombackstrom** [executablebooks/meta#932] [Q&A]
Hey,

  Had to reinstall my anaconda on Ubuntu today, but it refuses to install jupyter-book. That is, on a fresh install of anaconda, I did a "conda update -all" and then "conda install -c conda-forge jupyter-book". It reports that it found a conflict and goes into a forever-loop (more than 4 hours) trying to fix it. 
   Any ideas on how to resolve the issue? I'm the maintainer of a large jupyter-book project and am currently unable to compile it, so this is important. As a workaround, I could install it with pip, but mixing installation channels is usually a bad idea.

thanks,

Tom

**Comment by NickleDave** [comment]
what version of conda?

Just to rule this out:  
- [ ] are you installing into `(base)` or are you using an isolated environment?
- [ ] have you tried putting a lower bound on jupyter-book or pinning the version?

https://www.anaconda.com/blog/understanding-and-improving-condas-performance

Installing a specific version into an isolated environment will probably be many times faster than asking the dependency resolver to consider whether every possible verson of jupyter-book can be installed with every other package in the base environment. If you're already doing that, then there's some other issue but more info would be needed.

**Reply by chrisjsewell** [reply]
If you are not using mamba, I would strongly advise it

**Reply by chrisjsewell** [reply]
(https://github.com/mamba-org/mamba)

**Reply by tombackstrom** [reply]
Retried a few hours later and it worked. Weird. Thanks for the suggestions tho!

**Comment by NickleDave** [comment]
Also that 🙂 and of course you can now use their solver in conda https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community
